restli-server/src/main/java/com/linkedin/restli/server/RestLiConfig.java

```diff
@@ -593,4 +597,23 @@ public void setFillInDefaultValues(boolean fillInDefaultValues)
   {
     _fillInDefaultValues = fillInDefaultValues;
   }
+
+  /**
+   * Set a custom {@link MethodAdapterRegistry} in the config.
+   *
+   * @param methodAdapterRegistry a custom to be set in the config.
+   */
+  public void setMethodAdapterRegistry(MethodAdapterRegistry methodAdapterRegistry) {
```

Comment:
If we follow my advice about altering the naming, the naming here should be consistent. The exact naming is up for debate.